### PR TITLE
[samples] Remove duplicate samples

### PR DESCRIPTION
**Pull request by AlexJerabek**
Much of the content in Range samples was duplicated in the Basics page. This PR consolidates that information.

Note that this PR includes changes from PR #269.

### CODE REVIEW

File: docs/resources/samples/excel-samples.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ function main(workbook: ExcelScript.Workbook) {
 
 This sample finds the first PivotTable in the workbook and logs the values in the "Grand Total" cells (as highlighted in green in the image below).
 
-:::image type="content" source="../images/sample-pivottable-grand-total-row.png" alt-text="A PivotTable showing fruit sales with the Grand Total row highlighted green.":::
+:::image type="content" source="../../images/sample-pivottable-grand-total-row.png" alt-text="A PivotTable showing fruit sales with the Grand Total row highlighted green.":::
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/excel-samples.md:294]:
```suggestion
:::image type="content" source="../../images/sample-pivottable-grand-total-row.png" alt-text="A PivotTable showing fruit sales with the Grand Total row highlighted green":::
```

File: docs/resources/samples/excel-samples.md [hunk 2]
```diff
@@ -319,7 +319,7 @@ function main(workbook: ExcelScript.Workbook) {
 
 This script creates a drop-down selection list for a cell. It uses the existing values of the selected range as the choices for the list.
 
-:::image type="content" source="../images/sample-data-validation.png" alt-text="A worksheet showing a range of three cells containing color choices 'red, blue, green' and next to it, the same choices shown in a drop-down list.":::
+:::image type="content" source="../../images/sample-data-validation.png" alt-text="A worksheet showing a range of three cells containing color choices 'red, blue, green' and next to it, the same choices shown in a drop-down list.":::
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/excel-samples.md:322]:
```suggestion
:::image type="content" source="../../images/sample-data-validation.png" alt-text="A worksheet showing a range of three cells containing color choices 'red, blue, green' and next to it, the same choices shown in a drop-down list":::
```

File: docs/resources/samples/range-basics.md [hunk 3]
```diff
@@ -1,281 +0,0 @@
----
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/range-basics.md:1]: do we need a redirect for this file too?

File: docs/resources/samples/add-excel-comments.md [hunk 4]
```diff
@@ -67,4 +67,4 @@ function main(workbook: ExcelScript.Workbook) {
 
 ## Training video: Add comments
 
-[![Watch step-by-step video on how to add comments in an Excel file](../../images/comments-vid.jpg)](https://youtu.be/CpR78nkaOFw "Step-by-step video on how to add comments in an Excel file")
+[Watch Sudhi Ramamurthy walk through this sample on YouTube](https://youtu.be/CpR78nkaOFw).
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/add-excel-comments.md:70]: Could we use the video embed format? e.g. [!VIDEO https://www.youtube.com/embed/vPEqbb7t6-Y?start=154] (I got this from the "before" on docs/resources/scenarios/analyze-web-downloads.md)

Comment (ElizabethSamuel-MSFT) [inline docs/resources/samples/add-excel-comments.md:70]: Are you planning to delete the pictures no longer being used?

File: docs/resources/samples/excel-samples.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/resources/samples/excel-samples.md:294]: The alt-text periods were added on the accessibility advice of @lindalu-MSFT, though I'd love to double-check that I didn't misinterpret something.

File: docs/resources/samples/add-excel-comments.md (hunk 4 above)
Comment (AlexJerabek) [reply docs/resources/samples/add-excel-comments.md:70]: @mscharlock gave the guidance to not embed YouTube videos, as they do not meet accessibility criteria. Also, these videos are ostensibly from a third-party, so we did not want to promote them as first-party content.

Comment (AlexJerabek) [reply docs/resources/samples/add-excel-comments.md:70]: PR #269 addresses this. My apologies for merging the two here.

File: docs/resources/samples/range-basics.md (hunk 3 above)
Comment (AlexJerabek) [reply docs/resources/samples/range-basics.md:1]: Good catch.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Approving non-video changes as Michelle to review vid changes

**Comment by AlexJerabek** [comment]
Merging, but still open to feedback that can be addressed in subsequent PRs.

**Pull request merged**